sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -508,8 +504,8 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
       test("2016-03-13 03:00:00", "PST", "2016-03-13 10:00:00.0")
       test("2016-11-06 00:59:59", "PST", "2016-11-06 07:59:59.0")
       // 2016-11-06 01:00:00 PST could be 2016-11-06 08:00:00 UTC or 2016-11-06 09:00:00 UTC
-      test("2016-11-06 01:00:00", "PST", "2016-11-06 09:00:00.0")
-      test("2016-11-06 01:59:59", "PST", "2016-11-06 09:59:59.0")
+      test("2016-11-06 01:00:00", "PST", "2016-11-06 08:00:00.0")
```

Comment:
@cloud-fan @HyukjinKwon @gatorsmile Here is a behavior change comparing to the master. I will compare to 2.4. Maybe master and 2.4 behave differently here.

Reply:
This is the behavior change comparing to 2.4: https://github.com/apache/spark/blob/branch-2.4/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala#L580-L581

Reply:
I will restore previous behavior by reverting https://github.com/apache/spark/pull/27807/commits/52cdd206e9cddd5e86bd79f3e2141f5e9a437b35